CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 =======
 ## [Unreleased]
-- No changes yet.
+- api: expose `CallerPeerAddress` field in the transport request. Middleware and the server handler 
+  can then utilise this to implement audit, logging, various host based rate limiting, etc.
```

Comment:
(1) after the last incident custom rate limiting is deprecated, let's not mention it here.
(2) Any thoughts how this will interact with proxes in between? Per caller rate limiting will explicitly breaks since there's no guarantee that a single host-port maps to a single caller.

I'd love to hear @jronak thoughts on this PR :)

Reply:
Obviously, this will only reveal the identity of the immediate caller. in an SWN mesh like at Uber however, that identity corresponds to the host address hosting the caller service container in most cases. 

However, like in all networks, there is the possibility of the true address of the caller being obscured by relays and proxies, etc.